modules/oc-mirror-imageset-config-parameters-v2.adoc

```diff
@@ -0,0 +1,395 @@
+|The Operator package channel configuration.
+|Object
+
+|`mirror.operators.packages.channels.name`
+|The Operator channel name, unique within a package, to include in the image set.
+|String. For example: `fast` or `stable-v4.15`.
+
+|`mirror.operators.packages.channels.maxVersion`
+|The highest version of the Operator mirror across all channels in which it exists. 
+|String. For example: `5.2.3-31`
+
+|`mirror.operators.packages.channels.minVersion`
+|The lowest version of the Operator to mirror across all channels in which it exists. 
+|String. For example: `5.2.3-31`
+
+|`mirror.operators.packages.maxVersion`
+|The highest version of the Operator to mirror across all channels in which it exists. 
+|String. For example: `5.2.3-31`.
+
+|`mirror.operators.packages.minVersion`
+|The lowest version of the Operator to mirror across all channels in which it exists. 
+|String. For example: `5.2.3-31`.
+
+|`mirror.operators.packages.bundles`
+|The selected bundles configuration.
+|Array of objects. For example:
+
+[source,yaml,subs="attributes+"]
+----
+operators:
+  - catalog: registry.redhat.io/redhat/redhat-operator-index:vBranch Build
+    packages:
+    - name: 3scale-operator
+      bundles:
+      - name: 3scale-operator.v0.10.0-mas
+----
+
+|`mirror.operators.packages.bundles.name`
+|Name of the bundle selected for mirror (as it appears in the catalog)
+|String. Example : `3scale-operator.v0.10.0-mas`
+
+|`mirror.operators.targetCatalog`
+|An alternative name and optional namespace hierarchy to mirror the referenced catalog as.
+|String. For example: `my-namespace/my-operator-catalog`
+
+|`mirror.operators.targetCatalogSourceTemplate`
+|Path on disk for a template to use to complete catalogSource custom resource generated by oc-mirror
+|String. For example: `/tmp/catalog-source_template.yaml`
+Example of a template file: 
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: discarded
+  namespace: openshift-marketplace
+spec:
+  image: discarded
+  sourceType: grpc
+  updateStrategy:
+	registryPoll:
+  	interval: 30m0s
+----
+
+|`mirror.operators.targetTag`
+|An alternative tag to append to the `targetName` or `targetCatalog`.
+|String. For example: `v1`
+
+|`mirror.platform`
+|The platform configuration of the image set.
+|Object
+
+|`mirror.platform.architectures`
+|The architecture of the platform release payload to mirror.
+|Array of strings. For example:
+[source,yaml]
+----
+architectures:
+  - amd64
+  - arm64
+  - multi
+  - ppc64le
+  - s390x
+----
+
+The default value is `amd64`. The value `multi` ensures that the mirroring is supported for all available architectures, eliminating the need to specify individual architectures.
+
+|`mirror.platform.channels`
+|The platform channel configuration of the image set.
+|Array of objects. For example:
+
+[source,yaml,subs="attributes+"]
+----
+channels:
+  - name: stable-4.10
+  - name: stable-Branch Build
```

Comment:
why branch build here.

Reply:
It will be replaced with {product-version}